restlight-core/src/main/java/io/esastack/restlight/core/resolver/reqentity/FlexibleRequestEntityResolverFactory.java

```diff
@@ -131,7 +132,7 @@ private DefaultResolver(StringConverter converter,
             return checkRequired(nav, converter, Result.err());
         }
 
-        protected MediaType getMediaType(RequestEntity entity) {
+        protected MediaType getMediaType(RequestEntity entity, HttpRequest request) {
```

Comment:
Why add the request parameter to the method？

Reply:
I remove the unnecessary use in `RequestEntity` and there needs the request to get `format` parameter.